sync/sync_head.go

```diff
@@ -203,6 +203,8 @@ func (s *Syncer[H]) verify(ctx context.Context, newHead H) (bool, error) {
 // subjectiveHead.
 // A non-nil error is returned when networkHead can't be verified.
 func (s *Syncer[H]) verifyBifurcating(ctx context.Context, subjHead, networkHead H) error {
+	log.Warnw("sync: header bifurcation started", "height", networkHead.Height(), "hash", networkHead.Hash().String())
```

Comment:
nit: `sync:` is redundant as the logger's name already, so it will basically repeat.

Reply:
Done.